lib/common/huf.h

```diff
@@ -247,7 +247,7 @@ size_t HUF_readStats(BYTE* huffWeight, size_t hwSize,
 
 /** HUF_readCTable() :
  *  Loading a CTable saved with HUF_writeCTable() */
-size_t HUF_readCTable (HUF_CElt* CTable, unsigned* maxSymbolValuePtr, const void* src, size_t srcSize);
+size_t HUF_readCTable (HUF_CElt* CTable, unsigned* maxSymbolValuePtr, const void* src, size_t srcSize, unsigned *hasZeroWeights);
```

Comment:
minor point : I'm not too thrilled that the function signature had to be changed (and complexified) to support this new scenario, even though it's a valid solution.
I suppose an alternative could have been to scan the resulting `CTable` for potential presence of 0-freq slots.
It may be suspected that this alternative operation may cost more runtime, just it wasn't measured.